docs/sources/tempo/operations/cross_tenant_query.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Cross-tenant query federation
+menuTitle: Query
```

Comment:
If we say "Query" is this going to be confusing with queries used for TraceQL? What about updating it to say something like: 
```suggestion
menuTitle: Cross-tenant query
```